assets/js/eventCardFetcher.js

```diff
@@ -1,59 +1,61 @@
-const eventCard = require("./partials/event-card.hbs");
-
-$(function () {
-    const api = new GhostContentAPI({
-        url: ADVISORY.SITE_URL,
-        key: ADVISORY.CONTENT_API_KEY,
-        version: "v4",
-    });
+const api = new GhostContentAPI({
+    url: ADVISORY.SITE_URL,
+    key: ADVISORY.CONTENT_API_KEY,
+    version: "v4",
+});
 
-    const MONTHS = [
-        "January",
-        "February",
-        "March",
-        "April",
-        "May",
-        "June",
-        "July",
-        "August",
-        "September",
-        "October",
-        "November",
-        "December",
-    ];
+const MONTHS = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
 
-    const parseEvent = (post) => {
-        const splitTitle = post.title.split(":"); // <!> This assumes all event titles contain a colon
-        const splitDate = post.custom_excerpt.split("-");
-        return {
-            day: splitDate[2],
-            month: MONTHS[parseInt(splitDate[1]) - 1],
-            titlePrefix: splitTitle[0],
-            titleSuffix: splitTitle[1],
-        };
+const parseEvent = (post) => {
+    const splitTitle = post.title.split(":"); // <!> This assumes all event titles contain a colon
+    const splitDate = post.custom_excerpt.split("-");
+    return {
+        day: splitDate[2],
+        month: MONTHS[parseInt(splitDate[1]) - 1],
+        titlePrefix: splitTitle[0],
+        titleSuffix: splitTitle[1],
     };
+};
+
+const date = new Date().toISOString().split("T")[0];
 
-    const renderContainer = (posts) => {
-        if (posts.length > 0) {
-            $("#event-card-container").html(
-                eventCard({
-                    events: posts.slice(0, 4).map(parseEvent),
+window.eventCards = function () {
+    return {
+        posts: [],
+        showPlaceholder: true,
+        placeholderText: "Loading...",
```

Comment:
Since the value of this variable isn't used for the loading placeholder (as it's already specified in the HTML), can we set this to `null` instead to make it clearer?